packages/docs/getting-started/running-a-validator.md

```diff
@@ -299,30 +308,42 @@ In order to be elected as a Validator, you will first need to register your grou
 Register your Validator Group:
 
 ```bash
+# On your local machine
 celocli validatorgroup:register --from $CELO_VALIDATOR_GROUP_ADDRESS --commission 0.1
 ```
 
 Register your Validator:
 
 ```bash
+# On your local machine
```

Comment:
\# Note that because we have authorized a Validator signer, this step could also be performed on the Validator machine. Running it on the local machine allows us to avoid needing to install the celocli on the Validator machine